ee/localserver/request-controlservice.go

```diff
@@ -0,0 +1,21 @@
+package localserver
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func (ls *localServer) requestControlServiceFetchHanlder() http.Handler {
+	return http.HandlerFunc(ls.requestControlServiceFetchFunc)
+}
+
+func (ls *localServer) requestControlServiceFetchFunc(w http.ResponseWriter, r *http.Request) {
+	if ls.controlService == nil {
+		sendClientError(w, fmt.Sprintf("control service not configured"))
+		return
+	}
+
+	if err := ls.controlService.Fetch(); err != nil {
```

Comment:
I wonder if we'd also want a flag to make the control service make requests for each subsystem, like temporarily flipping `agent.Flags.ForceControlSubsystems`? Maybe depends on what this PR is needed for.

Reply:
There is a little more information about what this PR is for [here](https://github.com/kolide/launcher/issues/1086). I think that during the auth flow, if a user fixes an issue, we probably want to update the menu right meow. So the user doesn't still have a yellow / red icon on their task bar.

> I wonder if we'd also want a flag to make the control service make requests for each subsystem, like temporarily flipping agent.Flags.ForceControlSubsystems? Maybe depends on what this PR is needed for.

Do you mean like if a sub system was disabled, we want to for it to update?

Reply:
Ah ok, that makes sense and in that case we probably don't need the flag! I was thinking more along the lines of, "what if local data is bad/corrupt so we want to force launcher to refresh its data even if it doesn't think it's out-of-date", but I guess that'd be accomplished on the k2 side by regenerating hashes, maybe.

Reply:
I think the force rebuild is a good thing to consider. I'm not sure quote where it'll land. For the `flags`, it's easy for k2 to add a cache bust flag (it already has one)

But I'm less sure about the other systems.

Reply:
I agree -- it wouldn't belong in auth flow.